core/src/watchosMain/kotlin/com/powersync/DatabaseDriverFactory.watchos.kt

```diff
@@ -31,3 +31,9 @@ private val didLoadExtension by lazy {
 
     true
 }
+
+@ExperimentalPowerSyncAPI
+public actual fun resolvePowerSyncLoadableExtensionPath(): String? {
+    didLoadExtension
```

Comment:
It looks like this method can also throw if the extension has not been loaded correctly. It seems like the JVM version also throws in this case. Should we annotate the `resolvePowerSyncLoadableExtensionPath` method with `@Throws`?